src/duckling/game/drawable/tile-block-drawable-drawer.ts

```diff
@@ -0,0 +1,114 @@
+import {RenderTexture, Texture, Sprite, Graphics, DisplayObject, Container, BaseTexture} from 'pixi.js';
+
+import {Vector} from '../../math/vector';
+import {AssetService} from '../../project/asset.service';
+
+import {TileBlockDrawable, getTileWidth} from './tile-block-drawable';
+
+export function drawTileBlockDrawable(tileBlockDrawable : TileBlockDrawable, assetService : AssetService) : DisplayObject {
+    if (!tileBlockDrawable.textureKey) {
+        return null;
+    }
+
+    let baseTexture = assetService.get({key: tileBlockDrawable.textureKey, type: "TexturePNG"});
+    if (!baseTexture) {
+        return null;
+    }
+    
+    // let container = new Container();
+    return _constructTileBlockSprite(tileBlockDrawable, baseTexture, assetService);
+    // return container;
+}
+
+function _constructTileBlockSprite(tileBlockDrawable : TileBlockDrawable, baseTexture : BaseTexture, assetService : AssetService) : DisplayObject {
+    let tileSize = getTileWidth(tileBlockDrawable, assetService);
+    if (tileSize <= 0) throw new Error("Invalid tile size specified");
```

Comment:
You should use drawMissingAsset instead of throwing an error. The editor should not crash if a user types a negative number into the field. I would update the attribute's component so that it shows a warning if this happens.